src/Illuminate/Routing/RouteDependencyResolverTrait.php

```diff
@@ -41,12 +41,12 @@ public function resolveMethodDependencies(array $parameters, ReflectionFunctionA
 
         $values = array_values($parameters);
 
+        $shouldSkipValue = new \StdClass;
```

Comment:
Isn't its real name `stdClass`?

```suggestion
        $shouldSkipValue = new \stdClass;
```